src/opm/input/eclipse/EclipseState/Grid/NNC.cpp

```diff
@@ -17,6 +17,8 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 #include <array>
+#include <deque>
+#include <cstddef>
```

Comment:
Even though we get it through a transitive include path, you should probably `#include <algorithm>` explicitly here in order to get appropriate declarations for `std::sort()`/`std::stable_sort()`/`std::unique()`.

Reply:
done